CHANGELOG.md

```diff
@@ -117,7 +122,7 @@ All notable changes to this project will be documented in this file.
    - `set(text _text: String?, duration: TimeInterval)` in [[PR]](https://github.com/goktugyil/EZSwiftExtensions/pull/340) by *vilapuigvila*
 
 11. **UIImage**
-   - `var base64: String` in [[PR]](https://github.com/goktugyil/EZSwiftExtensions/pull/363) by *lfarah*## [Version 1.8] 
```

Comment:
Please revert this and all prior spaces.

Reply:
I'm Sorry, fixed.